bin/utils/java-utils.sh

```diff
@@ -63,8 +63,10 @@ validate_java_home() {
   fi
 
   # As we know the java -version command works then strip out the line we need
-  java_version_output=$(${JAVA_HOME}/bin/java -version 2>&1 | grep ^"java version")
-
+  java_version_output=$(${JAVA_HOME}/bin/java -version 2>&1 | grep ^"openjdk version")
+  if [[ $java_version_output == "" ]]; then
+    java_version_output=$(${JAVA_HOME}/bin/java -version 2>&1 | grep ^"java version")
+  fi
```

Comment:
doing the || instead of this works for me

Reply:
might be it works on linux not z/os?

Reply:
this syntax is necessary for z/OS and docker compatibility